### PR TITLE
feat: add welcome email to add user lambda

### PR DESCRIPTION
**Pull request by SEBRATHEZEBRA**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -2,4 +2,5 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  testRegex: '/src/.*.test.ts$',
```
Comment (mattzcarey) [inline jest.config.js:5]: What does the $ mean?

File: services/tests/EmailWorkerInteg.test.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+describe("CloudFlare email worker health test", () => {
+  test("Should return 200 after successfully hitting the CloudFlare health endpoint", async() => {
+    const res =  await fetch("https://worker-email-production.mattc-543.workers.dev/api/health", {
```
Comment (mattzcarey) [inline services/tests/EmailWorkerInteg.test.ts:3]: can you store this in an env file. This exposes the URL.

File: services/tests/jest.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  testRegex: '/services/tests/.*Integ.test.ts$',
```
Comment (mattzcarey) [inline services/tests/jest.config.js:5]: normally test files. will be emailWorker.integ.test.ts for example

File: services/web-app/README.md [hunk 4]
```diff
@@ -26,9 +26,11 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 Involves starting our SST dev environment.
 
 ```bash
-npx sst dev
+npx sst dev stage <stage>
```
Comment (mattzcarey) [inline services/web-app/README.md:29]: not -- stage?

File: services/web-app/src/functions/add-user/index.ts [hunk 5]
```diff
@@ -1,11 +1,30 @@
...
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "Authorization": await getVariableFromSSM(
+        process.env.CLOUDFLARE_WORKER_TOKEN_NAME ?? ""
+      ),
+    },
+    body: JSON.stringify({
+      "to": {"email": email, "name": name },
+      "from": { "email": "test@oriontools.ai", "name": "Orion Tools" },
```
Comment (mattzcarey) [inline services/web-app/src/functions/add-user/index.ts:21]: better to have it come from noreply@oriontools.ai. Name: 'Matt from Orion Tools'

File: services/web-app/sst.config.ts [hunk 6]
```diff
@@ -35,10 +34,11 @@ export default {
           consumer1: {
             function: {
               handler: "src/functions/add-user/index.main",
-              permissions: ["dynamodb"],
+              permissions: ["dynamodb", "ssm"],
               environment: {
-                STAGE: getStage(),
-              }
+                CLOUDFLARE_WORKER_URL: "https://worker-email-production.mattc-543.workers.dev/api/email",
```
Comment (mattzcarey) [inline services/web-app/sst.config.ts:39]: this should be a secret.

**Review by mattzcarey — CHANGES_REQUESTED** [review]

**Review by mattzcarey — APPROVED** [review]

File: services/web-app/src/functions/add-user/index.ts (hunk 5 above)
Comment (mattzcarey) [reply services/web-app/src/functions/add-user/index.ts]: can you change this

File: jest.config.js (hunk 1 above)
Comment (mattzcarey) [reply jest.config.js]: ?

File: .github/workflows/pr.yml [hunk 7]
```diff
@@ -16,6 +16,7 @@ jobs:
       BASE_SHA: ${{ github.event.pull_request.base.sha }}
       GITHUB_SHA: ${{ github.sha }}
       OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+      CLOUDFLARE_WORKER_HEALTH_URL: ${{ secrets.CLOUDFLARE_WORKER_HEALTH_URL }}
```
Comment (mattzcarey) [inline .github/workflows/pr.yml:19]: test

Comment (SEBRATHEZEBRA) [reply .github/workflows/pr.yml:19]: would you want me to add test to the environment variable name?

**Pull request merged**